lumigator/python/mzai/jobs/inference/model_clients.py

```diff
@@ -145,3 +146,31 @@ def _chat_completion(
             temperature=config.params.temperature,
             top_p=config.params.top_p,
         )
+
+
+class HuggingFaceModelClient(BaseModelClient):
+    def __init__(self, config: InferenceJobConfig):
```

Comment:
Maybe declare the types for _config and _pipeline here, specially _pipeline since it's a method in disguise :)

Reply:
Thanks Javier! Let's address than in the final version of the code.